HoloPipelines/pipelines/components/compNumpyTransformation.py

```diff
@@ -0,0 +1,47 @@
+import scipy.ndimage
+
+
+def crop(img, cropx, cropy, cropz):
```

Comment:
name is misleading as this does centering and not just cropping the edges.

Reply:
change crop[x,y,z] to something more explicit res[x,y,z]

Reply:
renamed to res()

Reply:
ah i see where the confusion came about. My bad i was initially referring to the arguments.